far/FarEng.hlf.m4

```diff
@@ -5555,75 +5555,87 @@ bits control the behavior of left and right key combinations.
 
 @Panel.ShellRightLeftArrowsRule
 $ #far:config Panel.ShellRightLeftArrowsRule#
- Параметр позволяет управлять поведением стрелок влево/вправо (как на основной, так и на дополнительной клавиатуре).
-
- Значения:
- 0 - ^<wrap>поведение как у 1.70: если командная строка непустая, то клавиши
-Left/Right и Num4/Num6 действуют по-разному в зависимости от
-режима панели: если имена файлов отображаются в несколько колонок
-(по умолчанию режимы 2 и 3), то команды вправо/влево применяются
-к панели (как и при пустой командной строке); а если имена файлов
-отображаются в одну колонку (по умолчанию все остальные режимы),
-то команды вправо/влево применяются к командной строке.
- 1 - ^<wrap>клавиши Left/Right и Num4/Num6 при включённой панели всегда
-применяются только к ней, независимо от содержимого командной
-строки и режима панели.
-
- Примечание: в командной строке есть CtrlD/CtrlS.
+ This Boolean parameter controls the behavior of left and right arrow
+keys, both on main keyboard and numeric keypad.
 
- По умолчанию значение = 0.
+ False - ^<wrap>As in Far 1.70. If command line is not empty, the
+behavior of #Left#, #Right#, #Numpad4#, and #Numpad6# keys depends
+on the ~panel view mode~@PanelViewModes@.
+         - ^<wrap>If file names are displayed in multiple stripes (panel
+modes 2 and 3 by default), then the keys move panel cursor, just like
+with empty command line.
+         - ^<wrap>If file names are displayed in a single stripe (all
+other panel modes by default), the keys control the command line caret.
+ True  - ^<wrap>When the panel is on, the #Left#, #Right#, #Numpad4#,
+and #Numpad6# keys always move panel cursor, even if the command line
+is not empty. The behavior does not depend on the current panel mode.
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ Note: The #Ctrl+D# and #Ctrl+S# keys always move command line caret.
+
+ Default value: False (traditional behavior).
+
+ This parameter can be changed via ~far:config~@FarConfig@ only.
 
 
 @Panel.Layout.ScrollbarMenu
-$ #far:config Panel.ShellRightLeftArrowsRule#
- Параметр разрешает показ полосы прокрутки в меню, если пунктов больше, чем высота меню. Если значение =0, то Far не будет отображать полосу прокрутки.
+$ #far:config Panel.Layout.ScrollbarMenu#
+ This Boolean parameter enables menu scrollbar when there are more menu
+items than can fit in menu window vertically.
```

Comment:
- Is `window` really needed here?
- article?

Reply:
Shortened to "than can fit vertically," both English and Russian. Please let me know if you disagree.